python/activity_stream/ui/note_widget.py

```diff
@@ -5,65 +5,96 @@
 #      by: pyside-uic 0.2.13 running on PySide 1.1.1
```

Comment:
Seems like if we're not generating these files from the .ui files they shouldn't be in the ui directory.